utils/rpc.mjs

```diff
@@ -0,0 +1,8 @@
+import { createSolanaRpc, createSolanaRpcSubscriptions_UNSTABLE } from "@solana/web3.js";
+
+export const rpc = createSolanaRpc(
+    process.env.RPC_ENDPOINT,
+);
+export const rpcSubscriptions = createSolanaRpcSubscriptions_UNSTABLE(
+    process.env.WS_ENDPOINT,
+);
```

Comment:
In general, it's good to share these as widely as possible so that they can maximally coalesce requests and subscriptions together.